services/src/functional-tests/README.md

```diff
@@ -0,0 +1,43 @@
+# Functional tests
+This module holds Functional tests designed to be run against staging OR virtual environment.
+
+# How to run 
+### Virtual (local Kilda)
+- Spawn your Kilda env locally by running
+```
+make build-latest 
+make up-test-mode
+```
+- Check your `kilda.properties`. It should point to your localhost environments.  
+`spring.profiles.active` should be set to `virtual`.
+- Check your `topology.yaml`. This is a file which will be used to spawn a virtual
+topology used by all the tests.
+- You can now run tests from IDE or run  
+`mvn clean test -Pfunctional`
+
+### Hardware (Staging)
+- Check your `kilda.properties`. It should point to your staging environments.  
+`spring.profiles.active` should be set to `hardware`.
+- Check your `topology.yaml`. It should represent your actual hardware topology.
+- You can now run tests from IDE or run  
```

Comment:
Please include step "mark `groovy` subdirectory in `functional-tests` module as a `test sources root` and ensure that `topology.yaml` and `kilda.properties` are present in the root of `functional-tests` to be able to run the tests from the IDE", that'll help new engineers to start debugging.

Reply:
fixed in #1236